### PR TITLE
Bugfix 20462

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

**Comment by montegoulding** [comment]
I've also found an odd behavior with option+down and opened a report about it as I can't see any code that's doing it http://quality.livecode.com/show_bug.cgi?id=20463

File: Documentation/guides/The LiveCode IDE.md [hunk 1]
```diff
@@ -586,11 +586,12 @@ placeholder fields which you can navigate with the tab key. Some
 completions may have multiple placeholders with the same name which
 will be edited together. Clicking within a placeholder will select it.
 
-The current automatic completion of control structures has not changed.
+#### Autocomplete Pro
 
-Autocomplete provides the following features:
+> **Note:** Autocomplete is available in LiveCode Indy and above
```
Comment (livecodeali) [inline Documentation/guides/The LiveCode IDE.md]: Should this say `Autocomplete Pro is available...`?

File: Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript [hunk 2]
```diff
@@ -2355,49 +2355,68 @@ on rawKeyDown pKey
...
+      if (pKey is kKeyUpArrow and \
+            (the eventCommandKey is down or \
+            the eventControlKey is down) and \
+            the eventShiftKey is up and \
+            the eventOptionkey is up) or \
+            (pKey is 65 and \
+            the eventCommandKey is up and \
+            the eventControlKey is down and \
+            the eventShiftKey is up and \
+            the eventOptionkey is up) then
```
Comment (livecodeali) [inline Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript:2376]: These `down`s and `up`s won't compile under strict compilation

Comment (livecodeali) [reply Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript:2376]: Hmm, it appears they do! Sorry :-)

File: Documentation/guides/The LiveCode IDE.md (hunk 1 above)
Comment (montegoulding) [reply Documentation/guides/The LiveCode IDE.md]: yes it should

**Comment by livecodeali** [comment]
@livecode-vulcan review ok aa85cd4

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok aa85cd47da88726158d69a5ed3d234d7f311c473

**Pull request merged**